<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cameras</class>
 <widget class="QWidget" name="Cameras">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>959</width>
    <height>655</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Distribución de cámaras</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>251</width>
     <height>612</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="configuration">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Introduzca los datos:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Ancho</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="rows"/>
    </item>
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Largo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="cols"/>
    </item>
    <item>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Foco de la cámara</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="focus"/>
    </item>
    <item>
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Número de cámaras</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="numCameras"/>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Peso de la distancia entre cámaras</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="distPower"/>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Peso de las feromonas</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="pheromPower"/>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Evaporación feromonas</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="evaporation"/>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Número de hormigas</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="numAnts"/>
    </item>
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Porcentaje de locas</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="numCrazy"/>
    </item>
    <item>
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Umbral mínimo de parada</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="minThreshold"/>
    </item>
    <item>
     <widget class="QPushButton" name="button">
      <property name="text">
       <string>START</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>289</x>
     <y>9</y>
     <width>631</width>
     <height>631</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="imagesLayout">
    <item>
     <widget class="QWidget" name="image" native="true"/>
    </item>
    <item>
     <widget class="QWidget" name="pheromones" native="true">
      <widget class="Line" name="line_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>629</width>
         <height>3</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>20</width>
     <height>631</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
